rpm/wpa_supplicant.spec

```diff
@@ -1,18 +1,25 @@
 Name:       wpa_supplicant
 
 Summary:    WPA/WPA2/IEEE 802.1X Supplicant
-Version:    2.10
+Version:    2.11
 Release:    1
-License:    GPLv2
+License:    BSD-3-Clause
```

Comment:
Should be BSD. See JB#50383

Reply:
Huh? If you want to suggest wpa-supplicant relicensing from bsd to gpl you should be asking upstream. I'm talking here only about the string used for referring to the license which should follow our current convention.

Reply:
What? Do you intentionally misread theese? I was talking about the generic recommendation on the BSD versioning in licenses, and how we use it  - our convention is yet again ambiguous. Read the commit message, I referred [to what Fedora uses](https://src.fedoraproject.org/rpms/wpa_supplicant/blob/rawhide/f/wpa_supplicant.spec#_13).

Reply:
Our license string convention is what Fedora used to have some time ago. It's up for consideration if we should switch again to what Fedora is doing now, but until we make a concious decision and start doing it everywhere, we should stick to what we have now. Thus BSD-3-Clause -> BSD for now.

Reply:
Changed and JB#62443 for changing to proper version of BSD.

Reply:
Fedora (and other such as Arch or SUSE) switched to SPDX tags long time ago. 
Fedora's wiki says: "The License: tag in the spec file Preamble must be populated with an SPDX license expression that consists of a Fedora-acceptable license, or multiple Fedora-acceptable licenses joined by AND and/or OR operators. We say that such a license expression is itself Fedora-acceptable."

Thous using simply BSD would be wrong according to the Fedora packaging guidelines.

Reply:
We are not Fedora.